Typecheck fails because it can't find definitions for cypress or @testing-library/cypress

### Have you experienced this bug with the latest version of the template?

Yes

### Steps to Reproduce

1. Install the indie stack as described by the README (make sure to run setup and install dependencies)
2. Run `npm run typecheck`

### Expected Behavior

The type check should run and succeed.

### Actual Behavior

The following error is produced:

```
error TS2688: Cannot find type definition file for '@testing-library/cypress'.
  The file is in the program because:
    Entry point of type library '@testing-library/cypress' specified in compilerOptions

  cypress/tsconfig.json:15:34
    15     "types": ["node", "cypress", "@testing-library/cypress"],
                                        ~~~~~~~~~~~~~~~~~~~~~~~~~~
    File is entry point of type library specified here.

error TS2688: Cannot find type definition file for 'cypress'.
  The file is in the program because:
    Entry point of type library 'cypress' specified in compilerOptions

  cypress/tsconfig.json:15:23
    15     "types": ["node", "cypress", "@testing-library/cypress"],
                             ~~~~~~~~~
    File is entry point of type library specified here.


Found 2 errors.
```

--- a/fly.toml
+++ b/fly.toml
@@ -3,6 +3,7 @@ app = "indie-stack-template"
 kill_signal = "SIGINT"
 kill_timeout = 5
 processes = []
+swap_size_mb = 512
 
 [experimental]
   allowed_public_ports = []

--- a/start.sh
+++ b/start.sh
@@ -5,13 +5,5 @@
 # run, which is why this file exists in the first place.
 # Learn more: https://community.fly.io/t/sqlite-not-getting-setup-properly/4386
 
-# allocate swap space
-fallocate -l 512M /swapfile
-chmod 0600 /swapfile
-mkswap /swapfile
-echo 10 > /proc/sys/vm/swappiness
-swapon /swapfile
-echo 1 > /proc/sys/vm/overcommit_memory
-
 npx prisma migrate deploy
 npm run start

